contracts/src/v0.8/automation/interfaces/StreamsLookupCompatibleInterface.sol

```diff
@@ -17,4 +17,17 @@ interface StreamsLookupCompatibleInterface {
     bytes[] memory values,
     bytes memory extraData
   ) external view returns (bool upkeepNeeded, bytes memory performData);
+
+  /**
+   * @dev this is a new, optional function in v2.1. It is meant to surface streams lookup errors.
```

Comment:
nit.

- use @notice as above function
- `this is a new, optional function in v2.1.` -> `this is a new, optional function in streams lookup`. (Streams lookup is independent of v2.1 and will be supported in future versions)